easybuild/easyconfigs/c/CMake/CMake-3.6.2-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.6.2'
```

Comment:
ah, most of those changes were made already, sorry >_<

still, this easyconfig is no longer needed?